src/main/java/eggventory/model/items/Stock.java

```diff
@@ -44,7 +44,7 @@ public Stock(String stockType, String stockCode, int quantity, String descriptio
         this.stockCode = stockCode;
         this.quantity = quantity;
         this.description = description;
-        this.loaned = 0;
+        this.loaned = 0; //TODO: Determine whether to remove this attribute or update it from the LoanList.
```

Comment:
Hmm, what's the purpose of it in stock, actually this needs to be there isn't it, in order to calculate the numAvailable when you do list stock all?